analysis_rsp.tex

```diff
@@ -0,0 +1,50 @@
+
+\subsubsection{Portal}
+
+\textbf{P-101}: User interface cone search on the object catalog, review the results overplotted on the HIPS map.
+
+\textbf{P-102}: Astronomical Data Query Language (ADQL) cone search on the object catalog with magnitude constraints, create a color-magnitude diagram in the results view.
+
+\textbf{P-103}: Create exploratory color plots of different trans-Neptunian object (TNO) populations and perform statistical analyses to determine potential correlations. 
+
+\textbf{P-201}: Perform an LSST light curve classification (e.g., stars vs quasars vs AGNs) using Portal tools, leveraging visualization features to distinguish populations.
+
+\textbf{P-301}: Query annual data releases and deepCoadds for overlaps with other datasets, and generate publication-quality plots of data and images.
```

Comment:
Re. "overlaps with other datasets", gives the impression that "other datasets" will be served and joinable via the RSP Portal (unclear). 

However, we do need image-based analysis use cases. E.g., to find LSST images that spatially / temporally overlap user-defined coordinates / timeframes; to visualize and interact (Zoom, pan, rescale, pixel stats, markup); to save the image.